plugins/modules/zos_operator.py

```diff
@@ -129,40 +169,189 @@ def run_module():
         new_params = parse_params(module.params)
         rc_message = run_operator_command(new_params)
         result["rc"] = rc_message.get("rc")
-        result["content"] = rc_message.get("message").split("\n")
+
+        result["content"] = []
+        short_str = []
+        ssctr = 0
+        tstr = rc_message.get("stdout")
+        if tstr is not None:
+            for s in tstr.split("\n"):
+                result["content"].append(s)
+                if ssctr < 5:
+                    short_str.append(s)
+                    ssctr += 1
+        ssctr = 0
+        tstr = rc_message.get("stderr")
+        if tstr is not None:
+            for s in tstr.split("\n"):
+                result["content"].append(s)
+                if ssctr < 5:
+                    short_str.append(s)
+                    ssctr += 1
+
+        result["content"].append("Ran" + rc_message.get("call"))
+        result["changed"] = False
+
+        if int(result["rc"]) == 0:
+            if len(short_str) > 2:
```

Comment:
We discussed this a few min ago but if you could add comments here as to why '2' is chosen, will help future reviews/developers

Reply:
agreed.